sherpa.spec

```diff
@@ -15,6 +15,18 @@ Requires: openloops
 %endif # isamd64
 %endif # islinux
 
+%if "%{?cms_cxx:set}" != "set"
```

Comment:
These are all redundant and not needed. We don't need this macro. Just directly put what's needed on the configure/make lines.

Btw, `-O2 -std=c++0x` is it required, or was it just copy & paste from somewhere?